src/lib.rs

```diff
@@ -172,6 +201,36 @@ where
         Ok(())
     }
 
+    /// set filter bandwidth
+    pub fn set_filter_bandwidth(&mut self, bandwidth: BANDWIDTH) -> Result<(), Mpu6050Error<E>> {
+        Ok(self.write_bits(
+            CONFIG::ADDR,
+            CONFIG::BANDWIDTH.bit,
+            CONFIG::BANDWIDTH.length,
+            bandwidth as u8,
+        )?)
+    }
+
+    /// get filter bandwidth
+    pub fn get_filter_bandwidth(&mut self) -> Result<BANDWIDTH, Mpu6050Error<E>> {
+        let bandwidth: u8 = self.read_bits(
+            CONFIG::ADDR,
+            CONFIG::BANDWIDTH.bit,
+            CONFIG::BANDWIDTH.length,
+        )?;
+
+        match bandwidth {
```

Comment:
shouldn't these be different bandwiths? also: since you're already explicitly defining the numeric values in the enum maybe you can use that to do the conversion (not sure what the best approach is, just found [this reddit thread](https://stackoverflow.com/questions/28028854/how-do-i-match-enum-values-with-an-integer))? or at least do it in a `TryFrom` implementation to have it a bit more separated?